plugins/modules/perlmod_install_info.py

```diff
@@ -0,0 +1,386 @@
+        - 'auto'
+        - 'true'
+        - 'false'
+        default: 'auto'
+    try_yum:
+        description: Specify whether to check for modules using the dnf package
+          manager. Defaults to C(true) if I(try_dnf=false) and C(yum) executable
+          is available.
+        type: str
+        choices:
+        - 'auto'
+        - 'true'
+        - 'false'
+        default: 'auto'
+    try_apt:
+        description: Specify whether to check for modules using the apt package
+          manager. Defaults to C(true) if C(apt-file) executable is available.
+        type: str
+        choices:
+        - 'auto'
+        - 'true'
+        - 'false'
+        default: 'auto'
+    try_cpanm:
+        description: Specify whether to check for modules using cpanm. Defaults
+          to C(true) if C(cpanm) executable is available.
+        type: str
+        choices:
+        - 'auto'
+        - 'true'
+        - 'false'
+        default: 'auto'
+    update:
+        description: Specify whether to update package manager databases before
+          searching.
+        type: bool
+        default: false
+
+author:
+    - Jonathan Kamens (@jikamens)
+'''
+
+EXAMPLES = r'''
+# Search and fail if the package can't be found
+- name: Search for Net::DNS if it isn't already installed
+  perlmod_install_info:
+    name: Net::DNS
+  register: perlmod_info
+  failed_when: perlmod_info.missing is defined
+
+- name: Search for two modules, even if they're already installed
+  perlmod_install_info:
+    name:
+    - URI
+    - WWW::Mechanize
+  try_installed: false
+  register: perlmod_info
+
+- name: install dnf packages identified by perlmod_install_info
+  dnf:
+    name: "{{perlmod_info.dnf}}"
+  when: perlmod_info.dnf is defined
+
+- name: install yum packages identified by perlmod_install_info
+  yum:
+    name: "{{perlmod_info.yum}}"
+  when: perlmod_info.yum is defined
+
+- name: install yum packages identified by perlmod_install_info
+  apt:
+    name: "{{perlmod_info.apt}}"
+  when: perlmod_info.apt is defined
+
+- name: install cpanm packages identified by perlmod_install_info
+  cpanm:
+    name: "{{item}}"
+  with_items: "{{perlmod_info.cpanm}}"
+  when: perlmod_info.cpanm is defined
+'''
+
+RETURN = r'''
+installed:
+    description: List of modules that are already installed
+    type: list
+    elements: str
+    returned: when I(try_installed) is C(true) and installed modules were found
+    sample: ['Net::DNS']
+dnf:
+    description: List of dnf requirements that should be installed to provide
+      at least some of the required Perl modules
+    type: list
+    elements: str
+    returned: when I(try_dnf) is C(true) and requested modules were found in dnf
+    sample: ['perl(Net::DNS)']
+yum:
+    description: List of yum requirements that should be installed to provide
+      at least some of the required Perl modules
+    type: list
+    elements: str
+    returned: when I(try_yum) is C(true) and requested modules were found in yum
```

Comment:
There are more of these than the one you changed.

Reply:
I'm sorry, but I've reviewed the current docs again and I don't see what else needs to be changed. Can you point at one example to guide me in the right direction? Again, I'm doing my best here to give you what you want, but it's just not clear to me what that is.